programs/creative building tools/README.md

```diff
@@ -7,19 +7,19 @@ An assortment of scarpet apps to help you with your creative builds. Some have g
 ### cover.sc
 #### By Firigion
 
-Lets you cover a set of blocks with a set of other blocks for ease of spawnproofing a decorative build. To use it, put the block to cover in your offhand and the block to cover it with in your main hand. To do a bunch of diferent blocks at a time, create pairs by palcing a shulker box in the main hand and another one in the off hand. Pairs will be created from matching slots in the boxes. For more info, see [this](https://github.com/Firigion/scarpets#cover).
```

Comment:
Different with one f

Reply:
Yeah missed that made the change thanks.